src/legacy/core_plugins/data/public/index_patterns/index_patterns_service.ts

```diff
@@ -51,7 +51,7 @@ import setupRouteWithDefaultPattern from 'ui/index_patterns/route_setup/load_def
 import { getFromSavedObject, isFilterable } from 'ui/index_patterns/static_utils';
 
 // IndexPattern, StaticIndexPattern, StaticIndexPatternField, Field
-import * as types from 'ui/index_patterns/index.d.ts';
+// import * as types from 'ui/index_patterns/index.d.ts';
```

Comment:
To keep things clean I've fixed this [in a separate PR](https://github.com/elastic/kibana/pull/35682), so you can just merge master and won't have to worry about it here.